lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -0,0 +1,14 @@
+module ShopifyAPI
+  class FulfillmentOrder < Base
+    def self.all(options = {})
+      order_id = options.dig(:params, :order_id)
```

Comment:
Should this raise if `order_id` is not passed in `options`?

Reply:
Absolutely.